packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -124,8 +132,18 @@ class CopyVmsModalBody extends BaseComponent {
               <SelectSr
                 disabled={copyMode !== 'fullCopy'}
                 onChange={this.linkState('sr')}
+                predicate={
+                  this.isCurrentPlanHigherThanStarter
+                    ? undefined
+                    : this.getSrPredicate()
```

Comment:
I'd put this check in `getSrPredicate` so that the whole filtering logic is in the same place.